eventmesh-storage-plugin/eventmesh-storage-standalone/src/test/java/org/apache/eventmesh/storage/standalone/admin/StandaloneAdminTest.java

```diff
@@ -90,7 +90,7 @@ public void testDeleteTopic() {
 
     @Test
     public void testGetEvent() throws Exception {
-        Mockito.when(standaloneBroker.checkTopicExist(TEST_TOPIC)).thenReturn(TOPIC_EXISTS);
+        Mockito.when(standaloneBroker.checkTopicExist(TEST_TOPIC)).thenReturn(Boolean.TRUE);
         Mockito.when(standaloneBroker.getMessage(TEST_TOPIC, OFF_SET)).thenReturn(createDefaultCloudEvent());
```

Comment:
How about adding a `Topic Exists` comment here to compensate the removed boolean member? And a `Topic does not Exist` for L102.

Reply:
Not sure I understood, checkTopicExist function isn't already conveying exist or not exist in the name?  But I can add a comment no problem

Reply:
Okay, then let's not adding it.